<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature Sensor 1</string>
    </property>
    <widget class="QTextEdit" name="txt_TempSensor1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>104</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_EnAlarm1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Alarm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>63</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSpinBox" name="spb_TempHighThres1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_TempLowThres1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature High Threshold [°C]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature Low Threshold [°C]</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alm_TempHigh1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>45</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alm_TempHigh1_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_ReadTemp1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>491</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature Sensor 2</string>
    </property>
    <widget class="QTextEdit" name="txt_TempSensor2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>104</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable Alarm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>63</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>241</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSpinBox" name="spb_TempHighThres2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spb_TempLowThres2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature High Threshold [°C]</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature Low Threshold [°C]</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alm_TempHigh1_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="alm_TempHigh1_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>31</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Alarm</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_ReadTemp2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Read</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Sample Time (sec)</string>
    </property>
    <widget class="QComboBox" name="cbb_SampleTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>171</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select Sample Time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_Setting">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Connect</string>
    </property>
    <widget class="QComboBox" name="cbb_COM">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select COM</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbb_Baudrate">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Select Baudrate</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>COM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Baudrate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Connect">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Refresh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Received</string>
    </property>
    <widget class="QTextEdit" name="txt_DataReceived">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ClearData">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>270</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ReadData">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Notification</string>
    </property>
    <widget class="QTextEdit" name="txt_Notification">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ClearNoti">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
